html/pages/index.html

```diff
@@ -1,4 +1,198 @@
+        A fromer Special Forces operative turned mercenary is subjeced to rouge
+        experiment that leaves him with accelerated healing powers, adopting
+        the alter ego Deadpool
+      </p>
+
+      <footer class="featured-movie-meta">
+        R | 180 min | Action,Adventure,Comedy |
+        <time datetime="12-02-2016"> 12 February 2016</time>
+      </footer>
+    </section>
+  </article>
+</header>
+
+<section class="popular-movies" id="movies">
+  <a href="#">
+    <h3>Popular Movies</h3>
+  </a>
+
+  <a href="#">
+    <h3 class="all-movies">View All movies</h3>
+  </a>
+
+  <figure class="popular-movies-item">
+    <a href="#">
+      <img src="{{root}}/images/the-martian-preview.jpg"
+           alt="The Martian">
+    </a>
+  </figure>
+
+  <figure class="popular-movies-item">
+    <a href="#">
+      <img src="{{root}}/images/inside-out-preview.jpg"
+           alt="Inside out">
+    </a>
+  </figure>
+
+  <figure class="popular-movies-item">
+    <a href="#">
+      <img src="{{root}}/images/jurassic-world-preview.jpg"
+           alt="Jurassic world">
+    </a>
+  </figure>
+
+  <figure class="popular-movies-item">
+    <a href="#">
+      <img src="{{root}}/images/mad-max-preview.jpg"
+          alt="Mad Max">
+    </a>
+  </figure>
+
+  <figure class="popular-movies-item">
+    <a href="#">
+      <img src="{{root}}/images/star-wars-force-awakens-preview.jpg"
+           alt="Stra Wars force awakens">
+    </a>
+  </figure>
+
+  <figure class="popular-movies-item">
+    <a href="#">
+      <img src="{{root}}/images/avengers-age-of-ultron-preview.jpg"
+           alt="Avengers age of ultron">
+    </a>
+  </figure>
+</section>
+
+<section class="popular-shows" id="tv-shows">
+  <a href="#">
+    <h3>Popular TV Shows</h3>
+  </a>
+
+  <a href="#">
+    <h3 class="all-shows">View All TV sows</h3>
+  </a>
+
+  <figure class="popular-shows-item">
+    <a href="#">
+      <img src="{{root}}/images/walking-dead-preview.jpg"
+          alt="Walking dead">
+    </a>
+  </figure>
+
+  <figure class="popular-shows-item">
+    <a href="#">
+      <img src="{{root}}/images/game-of-thrones-preview.jpg"
+          alt="Game of thrones">
+    </a>
+  </figure>
+
+  <figure class="popular-shows-item">
+    <a href="#">
+      <img src="{{root}}/images/breaking-bad-preview.jpg"
+           alt="Breaking Bad">
+    </a>
+  </figure>
+
+  <figure class="popular-shows-item">
+    <a href="#">
+      <img src="{{root}}/images/empire-preview.jpg"
+           alt="Empire">
+    </a>
+  </figure>
+
+  <figure class="popular-shows-item">
+    <a href="#">
+      <img src="{{root}}/images/master-of-none-preview.jpg"
+           alt="Master of none">
+    </a>
+  </figure>
+
+  <figure class="popular-shows-item">
+    <a href="#">
+      <img src="{{root}}/images/mr-robot-preview.jpg"
+           alt="Mr robot">
+    </a>
+</figure>
+
+<section class="subscription">
+  <form>
+    <legend class="best-movies-week">
```

Comment:
What does the class name mean? Think about tags as about containers with a specific purpose. They can have different content, but purpose is always the same. So in order to be reliable the class names should display the purpose of the tag, not the content.

Bad: `best-movies-week`
Good: `subscription-form-description`